semanticdb/scalac/library/src/main/scala-2.13.2/scala/meta/internal/semanticdb/scalac/VersionCompilerOps.scala

```diff
@@ -0,0 +1,11 @@
+package scala.meta.internal.semanticdb.scalac
+
+import scala.tools.nsc.interactive.Global
+
+trait VersionCompilerOps {
+
+  def forceWarnings(global: Global) = {
+    global.runReporting.reportSuspendedMessages()
```

Comment:
This is the main thing that was needed to be added for some of the Scala versions.